new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java

```diff
@@ -329,7 +438,33 @@ private static void createDomainCrAndVerify(String adminSecretName,
     createDomainAndVerify(domain, domainNamespace);
   }
 
-  private void verifySearchResults(String queryCriteria, String regex,
+  private void verifyServerRunningInSearchResults(String serverName) {
+    int i = 0;
+    String queryResult = null;
+    String queryCriteria = "/_search?q=log:" + serverName;
+    while (i < maxIterationsPod) {
```

Comment:
variable name maxIterationsPod is not appropriate here, also use awaitility api like we do in other cases ...

Reply:
changed